virttest/kernel_interface.py

```diff
@@ -82,7 +82,7 @@ def proc_fs_value(self):
         try:
             return int(self.fs_value)
         except ValueError:
-            return self.fs_value
+            return self.fs_val
```

Comment:
Do you need add self.fs_val in init function?

Reply:
> Do you need add self.fs_val in init function?

I think no need,  the function `fs_value` do it.

Reply:
I see. Had better update other places as well, like L98 and L154.

Reply:
> I see. Had better update other places as well, like L98 and L154.

We can't do that,  L98 and L154 is to check if the new value be setted successfully ~

Reply:
Yeah, you are right.